security-analysis/security-analysis-api/src/test/java/com/powsybl/security/detectors/DefaultLimitViolationDetectorTest.java

```diff
@@ -147,14 +176,15 @@ void detectPermanentApparentPowerLimitOnSide2OfLine1() {
         line1 = network.getLine("NHV1_NHV2_1");
         line2 = network.getLine("NHV1_NHV2_2");
 
-        detector.checkPermanentLimit(line1, Branch.Side.TWO, 1101, violationsCollector::add, LimitType.APPARENT_POWER);
+        detector.checkPermanentLimit(line1, Branch.Side.TWO, 1.0f, 1101, violationsCollector::add, LimitType.APPARENT_POWER);
 
         Assertions.assertThat(violationsCollector)
                   .hasSize(1)
                   .allSatisfy(l -> {
                       assertEquals(1100, l.getLimit(), 0d);
                       assertEquals(1101, l.getValue(), 0d);
                       assertSame(Branch.Side.TWO, l.getSide());
+                      assertEquals(PERMANENT_LIMIT_NAME, l.getLimitName());
```

Comment:
maybe you can check the limit reduction here too

Reply:
yes done